optimizely/optimizely_config.py

```diff
@@ -12,13 +12,15 @@
 # limitations under the License.
 
 import copy
+from .helpers.condition import ConditionOperatorTypes
 
 from .project_config import ProjectConfig
 
 
 class OptimizelyConfig(object):
     def __init__(self, revision, experiments_map, features_map, datafile=None,
-                 sdk_key=None, environment_key=None, attributes=None, events=None):
+                 sdk_key=None, environment_key=None, attributes=None, events=None,
+                 audiences=None, delivery_rules=None):
```

Comment:
Watch for indentation here

Reply:
The lines above are indented as per Lint requirement for line breaking when too long.